src/EnergyPlus/EvaporativeCoolers.hh

```diff
@@ -307,6 +307,7 @@ namespace EvaporativeCoolers {
         int UnitLoadControlLimitsErrorIndex;  // root solver errors, art load ratio limits exceeded.
         int ZonePtr;                          // pointer to a zone served by an evaportive cooler unit
         int HVACSizingIndex;                  // index of a HVACSizing object for an evaportive cooler unit
+        Real64 ShutOffRelativeHumidity;       // Zone relative humidity above which the evap cooler is shut off.
```

Comment:
Although the IDD object inputs the RH as a percent, I would have preferred we keep it as a fraction internally, to make the calculations one step easier.  I'm not going to request that change now because it looks like this could be done or almost done.  But in general, I think percentages should just be a user-facing thing and keep things in fractional form internally where reasonable.

Reply:
make sense.